test/test_literal.py

```diff
@@ -98,3 +100,26 @@ class Foo(TestSetup):
     )
     with raises_invalid_choice():
         assert Foo.setup(f"--values {incorrect_value}")
+
+
+from typing import Literal
+from dataclasses import dataclass
+from simple_parsing import ArgumentParser
+
+
+@dataclass
+class SomeFoo(TestSetup):
+    param: Literal["bar", "biz"] = "biz"
+
+
+@pytest.mark.skipif(sys.version_info < (3, 9), reason="Bug is only in 3.9")
```

Comment:
I haven't checked, but since this bug is from a Python < 3.10 condition in `typing_extensions`, it probably also shows up in Python 3.8!

Reply:
Hmm I'll double-check, thanks!

Reply:
From my limited testing, it doesn't seem to be there with Python 3.8

Reply:
Did you upgrade `typing_extensions`?

On `typing_extensions==4.6.3`, with Python 3.8:
![image](https://github.com/lebrice/SimpleParsing/assets/6992947/dd7b2223-7219-4f10-80bc-5c41c0d88a19)